Wflow/src/standard_name.jl

```diff
@@ -203,6 +203,20 @@ const sbm_standard_name_map = Dict{String, NamedTuple}(
         (lens = @optic(_.land.soil.variables.zi), unit = "mm"),
     "soil_surface__temperature" =>
         (lens = @optic(_.land.soil.variables.tsoil), unit = "ᵒC"),
+    "land_surface__temperature" =>
+        (lens = @optic(_.land.land_surface_temperature.variables.LST), unit = "ᵒC"),
+    "land_surface__net_shortwave_radiation" =>
+        (lens = @optic(_.land.land_surface_temperature.variables.RSN), unit = "W m-2"),
+    "land_surface__net_longwave_radiation" =>
+        (lens = @optic(_.land.land_surface_temperature.variables.RLN), unit = "W m-2"),
+    "land_surface__net_radiation" =>
+        (lens = @optic(_.land.land_surface_temperature.variables.Rnet), unit = "W m-2"),
+    "land_surface__latent_heat_flux" =>
+        (lens = @optic(_.land.land_surface_temperature.variables.LE), unit = "W m-2"),
+    "land_surface__sensible_heat_flux" =>
+        (lens = @optic(_.land.land_surface_temperature.variables.H), unit = "W m-2"),
+    "land_surface__aerodynamic_resistance" =>
+        (lens = @optic(_.land.land_surface_temperature.variables.ra), unit = "s m-1"),
```

Comment:
Some suggested improvements for these standard names, also including a quantity name.

```suggestion
    "land_surface_radiation~net~shortwave__energy_flux" =>
        (lens = @optic(_.land.land_surface_temperature.variables.RSN), unit = "W m-2"),
    "land_surface_radiation~net~longwave__energy_flux" =>
        (lens = @optic(_.land.land_surface_temperature.variables.RLN), unit = "W m-2"),
    "land_surface_radiation~net~total__energy_flux" =>
        (lens = @optic(_.land.land_surface_temperature.variables.Rnet), unit = "W m-2"),
    "land_surface_air_heat~latent~evaporation__energy_flux" =>
        (lens = @optic(_.land.land_surface_temperature.variables.LE), unit = "W m-2"),
    "land_surface_air_heat~net~sensible__energy_flux" =>
        (lens = @optic(_.land.land_surface_temperature.variables.H), unit = "W m-2"),
    "atmosphere_bottom_air__bulk_momentum_aerodynamic_resistance" =>
        (lens = @optic(_.land.land_surface_temperature.variables.ra), unit = "s m-1"),
```